pkg/cleaner/azure/resourcegroup.go

```diff
@@ -95,3 +96,8 @@ func (c Cleaner) groupHasActivity(ctx context.Context, group resources.Group, si
 	// NotDone returns true when eventIter contains events.
 	return eventIter.NotDone(), nil
 }
+
+// isTerraformCIResourceGroup check if resource group name was created by Terraform CI.
+func isTerraformCIResourceGroup(s string) bool {
+	return strings.HasPrefix(s, "e2e") && len(s) == 7
```

Comment:
Here I check for `e2e` name prefix but also for the name length, since `e2e` might collide with "regular" cluster names.